data/human/transport missions.txt

```diff
@@ -3582,3 +3582,732 @@ mission "Hauler VI cargo prototype 3"
+
+			`	"And we should resist the temptation of hanging around the Alnitak system. There is a Navy base on Farpoint and even though they are Oathkeepers they still might scan us. Lucky for us I know a couple uninhabited systems with good solar wind and few pirates.  Well, we better get moving or we may never leave this garden."`
+				accept
+
+			label notyet
+			`	"We definitely need to do that before we go. Imagine if we got to Freedom only to realize we forgot something. Remember, we need one hundred Laser Rifles and ten boxes of Fragmentation Grenades. It should go without saying that we will be in trouble if we get caught with all this stuff so we probably shouldn't go through Republic space. That means we are going to need a lot of spare fuel to make it through all of the uninhabited systems north of here. Some friends of mine got us a Ramscoop, so make sure to install it. Also, if you are using a Scram Drive you might want to consider switching it out for a normal one to save on fuel and space."`
+
+			`	"And we should resist the temptation of hanging around Alnitak. There is a Navy base on Farpoint and even though they are Oathkeepers they still might scan us. Lucky for us I know a couple of systems with good solar wind and few pirates.  Well, we better get moving or we will get stuck in this garden forever."`
+				accept
+
+
+	on complete
+		outfit "Laser Rifle" -100
+		outfit "Fragmentation Grenades" -10
+		payment 200000
+		conversation
+			`You land near the factory and a crew comes out to unload the weapons and equipment. As you step outside you see the escorts have joined the meager 'Freedom Fighters' fleet overhead, ready for any pirate attack. Two people run up filled with excitement and embrace not only Beto but Ale and Bunker too. They seem to already know each other. "Lalo, Larisa. This is <first>. We couldn't have done it without them," says Beto by means of introduction.`
+			`	"Is that so?", says Larisa. "Thank you so much. You must stay for dinner!"`
+			choice
+				`	"I really ought to be going."`
+					goto getgoing
+				`	"I would love to."`
+			`	They give you a tours of the factory as equipment is moved in piece by piece. Part of the factory has been converted into a kitchen and social space. The smell of their cooking makes your belly rumble. "Thanks to you we finally have some means of production," says Larisa. "We are starting off with making components for Megaparcec. The unions there are especially desperate for new trade partners. That will give them more leverage and autonomy from the Syndicate. And that's good for us because we need all the allies we can get."`
```

Comment:
```suggestion
			`	They give you a tour of the factory as equipment is moved in piece by piece. Part of the factory has been converted into a kitchen and social space. The smell of their cooking makes your belly rumble. "Thanks to you we finally have some means of production," says Larisa. "We are starting off with making components for Megaparcec. The unions there are especially desperate for new trade partners. That will give them more leverage and autonomy from the Syndicate. And that's good for us because we need all the allies we can get."`
```